content/posts/fpcast.md

```diff
@@ -0,0 +1,478 @@
+---
+title: "Pyodide Function Pointer Cast Removal"
+date: 2021-12-06T20:27:27-08:00
+draft: true
+tags: ["internals"]
+author: "Hood Chatham"
+# author: ["Me", "You"] # multiple authors
+showToc: true
+TocOpen: false
+draft: false
+hidemeta: false
+comments: false
+description: "Why old versions of Pyodide had a low recursion limit and how version 0.19.0 supports a much higher one."
+# canonicalURL: "https://canonical.url/to/page"
+disableHLJS: true # to disable highlightjs
+disableShare: false
+hideSummary: false
+searchHidden: true
+ShowReadingTime: true
+ShowBreadCrumbs: true
+ShowPostNavLinks: true
+cover:
+    image: "<image path/url>" # image path/url
+    alt: "<alt text>" # alt text
+    caption: "<text>" # display caption under cover
+    relative: false # when using page bundles set this to true
+    hidden: true # only hide on current single page
+---
+
+In Pyodide v0.19, we can finally support the Python default recursion limit of
```

Comment:
If this is published before the release (and I think it would make sense to do it before) https://github.com/pyodide/pyodide-blog/wiki/Publication-schedule it would be good to indicate that the 0.19. release has not yet happened.